windows/win_unzip.ps1

```diff
@@ -57,17 +57,33 @@ $recurse = ConvertTo-Bool (Get-AnsibleParam -obj $params -name "recurse" -defaul
 $rm = ConvertTo-Bool (Get-AnsibleParam -obj $params -name "rm" -default "false")
 
 If ($ext -eq ".zip" -And $recurse -eq $false) {
-    Try {
-        $shell = New-Object -ComObject Shell.Application
-        $zipPkg = $shell.NameSpace([IO.Path]::GetFullPath($src))
-        $destPath = $shell.NameSpace([IO.Path]::GetFullPath($dest))
-        # 20 means do not display any dialog (4) and overwrite any file (16)
-        $destPath.CopyHere($zipPkg.Items(), 20)
-        $result.changed = $true
+    If (-Not (Get-WindowsFeature -Name 'Server-Gui-Shell')) {
```

Comment:
Need to check for windows version ([System.Environment]::OSVersion.Version) or possibly even better, use "Get-Command" to check if Get-WindowsFeature exists.  If it doesn't exist, you can probably also safely say it's not going to be core